$OpenBSD: patch-debugger_debmain_ml,v 1.1.1.1 2011/01/15 13:07:50 jasper Exp $

- Removed in new lablgtk2 version.
- Looks like it was MacOS-specific anyway.

--- debugger/debmain.ml.orig	Thu Jan 13 02:26:02 2011
+++ debugger/debmain.ml	Thu Jan 13 02:56:51 2011
@@ -271,7 +271,7 @@ class main_window () =
     else begin
       let about_item = GMenu.menu_item ~label:"About Obdb" () in
       let _ = about_item#connect#activate self#about in
-      GMain.set_platform_menubar menubar about_item
+      ()
     end;
 
     ignore (peer#event#connect#delete 
